napari/layers/image/image.py

```diff
@@ -729,11 +771,11 @@ def _set_view_slice(self):
                 self.level_shapes[self._thumbnail_level, not_disp] - 1,
             )
             indices[not_disp] = downsampled_indices
-
-            thumbnail_source = self.data[self._thumbnail_level][tuple(indices)]
+            thumb_indices = self._permute_indices(indices)
+            thumbnail_source = self.data[self._thumbnail_level][thumb_indices]
         else:
             self._transforms['tile2data'].scale = np.ones(self.ndim)
-            image_indices = self._slice_indices
+            image_indices = self._permute_indices(self._slice_indices)
```

Comment:
If I understand correctly, `_slice_indices` has already absorbed any permutation, so is `_permute_indices` inverting that so we can index using the non-permuted dimensions?

Reply:
Does this only need to happen for images? Have you tested with other layer types (e.g. points)?

Reply:
I think there is a bug here in the multiscale case (I'm pretty sure it should be `image = self.data[level][image_indices]` rather than `image = self.data[level][indices]`). I will add test coverage for permutations when the data is multiscale.